### ISSUE TITLE
openhaystack & esp32_ble_tracker

### ISSUE THREAD

**Issue by njwh1976** [barrenechea/esphome-config-files#2]
Is there a possibility to have these work at the same time?

INFO ESPHome 2023.8.2
INFO Reading configuration test_haystack.yaml...
Failed config

openhaystack: [source test_haystack.yaml:45]

  Component openhaystack cannot be used together with component esp32_ble_tracker.
  key: !secret haystack

**Comment by barrenechea** [comment]
@njwh1976 I don't think it would work, at least it didn't for me when I made the `openhaystack` component.

I believe there is a limitation over ESPHome side regarding multiple BLE-associated components running at the same time. Sorry!
